### DISCUSSION TITLE
Connecting a Graphql and  postgres database locally with Prisma

### DISCUSSION THREAD

**Discussion by binarydevelop** [prisma/prisma#10505] [Q&A] [answered]
I am trying to connect prisma to my local postgres database.   
What I am able to do? 
- I can run migrations using ``npx prisma migrate``
- It does create the table in database.

What I am not able to do?
- I cannot run queries on graphql playground 

It gives me an error message like this just after starting the playground at ``localhost:4466``
```js
{
  "errors": [
    {
      "message": "Project not found: 'default$default'",
      "code": 3016,
      "requestId": "local:api:ckwlvl3mz00010933yqxvgjrp"
    }
  ]
}
```
Also on the playground address bar i see ``Server cannot be reached``.
Here is my compose file which i am using 
```docker
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.8.4
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: postgres
            host: host.docker.internal
            port: 5432
            database: graphql_prisma
            user: postgres
            password: Tushar21
            migrations: true
            active: true
```
Here is the database url in my ``.env`` file  
```js
DATABASE_URL="postgresql://postgres:Tushar21@localhost:5432/graphql_prisma?schema=public"
```

What am i doing wrong?

**Comment by austincrim** [comment, marked as answer]
Hey @binarydevelop 👋,

Since you are able to connect to your DB through the Prisma CLI, it seems you have things set up correctly as far as Prisma is concerned. This seems more like a GraphQL setup error.

If you give me more context around your server setup, I can try to help, but I'm not a GraphQL expert.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
